add enum autocomplite docs and example (#235)
eclipsesource/jsonforms2-website: content/examples/autocomplete-enum.mdx

---
id: autocomplete
title: Autocomplete Renderer Example
description: Autocomplete Renderer Example
sidebar_label: Autocomplete Renderer
---

import { AutocompleteEnum, AutocompleteOneOf } from '../../src/components/docs/multiple-choice';

> The following examples display the autocomplete renderer.  A autocomplete renderer can be achieved by using an `enum` or an `oneOf` in the JSON schema and setting the `autocomplete` option in the UI schema.

Reviewer: ```suggestion
The following examples display the autocomplete renderer. 
The autocomplete renderer is used by specifying an `enum` or an `oneOf` in the JSON schema and setting the `autocomplete` option in the UI schema.
```
